composer/dag_test_utils/MAINTAINING.md

```diff
@@ -0,0 +1,38 @@
+# Maintenance Guide
+
+## How to release a new version
+* Make relevant code changes
+* Increment the version number in [setup.py](./setup.py) following [semver](https://semver.org/)
+* Add changes to the [CHANGELOG](./CHANGELOG.md)
+* If any usage info has changed, update the [README](./README.md)
+* [Test the distribution locally](#how-to-test-the-distribution-locally) in the [workflows directory](../workflows)
+* Open a PR and request reviews from a [Python Samples owner](https://github.com/orgs/GoogleCloudPlatform/teams/python-samples-owners) and a [Composer Codeowner](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/.github/CODEOWNERS#L24)
+* Once the PR is approved and merged, [push the new version to PyPI](#how-to-push-the-new-version-to-pypi)
+
+## How to test the distribution locally
+* In a `virtualenv`, run `pip install build`
+* Run `python -m build` to build the package. It will create a `dist/` directory containg a wheel (`.whl`) and a tar (`.tar.gz`). 
+* Change the [`requirements-test.txt`](../workflows/requirements-test.txt) file in the [`workflows`](../workflows) directory to import your distribution relatively, ensuring that `x.y.z` is replaced with your version number
+
+```
+../dag_test_utils/dist/cloud_composer_dag_test_utils-x.y.z.tar.gz
+```
+
+* Run `nox -s py-3.8 -- quickstart_test.py` (or the entire nox session, if you prefer) - if it passes without an `ImportError`, congrats! You've done it! 
+
+
+## How to push the new version to PyPI
```

Comment:
nit here:  this section is releasing instructions, and generally only relevant to internal teams with access (e.g. external users can't even mail the groupname).  Do we have a better internal place for recording per-repo information like accounts pypi accounts, how to retrieve secrets, etc?

Reply:
Great nit - I will ask Meredith who I know has been thinking about this.

Reply:
Per our offline convo - this will stay in for now, but will be migrated to internal team docs and replaced with a link to those internal docs 😄